Cut marks

leandrodreamer opened the pull request:
burn effect for the floor

https://user-images.githubusercontent.com/26260718/112695352-e88d4680-8e51-11eb-8e36-8e6acafba0a8.mp4

(still need testing in quest 1)

NeoSpark314 commented on game/BeepSaberMainMenu.gd:
```diff
@@ -60,8 +60,8 @@ func _load_playlists():
 	
 	if (!_playlists):
 		vr.log_error("No songs found in " + bspath);
-		for b in $SongsMenu/Songs.get_children():
```
Is this an intended change for this PR? What does it do?

leandrodreamer replied:
oh yea forgot abt that, it tried to clean the old list menu giving an error when there is no songs, i'll replace it by the clean function of the new itemlist that is used
so is just a little bugfix

NeoSpark314 commented:
This looks great! I will test it on Quest 1. If it turns out to be problematic from a performance perspective we can still make it an option and only turn it on by default when not running on quest 1

leandrodreamer commented:
i replaced that commented loop for a clear function witch will do the intended action for the new menu list
i also added a little change so the songs load with a default icon and then replace it by the real one, this is to prevent the buttons from changing size while loading, witch is pretty annoying

NeoSpark314 commented:
Very nice! Thanks for the contribution.

hankedan000 commented:
> This looks great! I will test it on Quest 1. If it turns out to be problematic from a performance perspective we can still make it an option and only turn it on by default when not running on quest 1

@NeoSpark314 Curious how you can detect which quest headset is in use? A while back, I was working on adding the quest 2 controllers to the devkit, but wasn't sure how to toggle them at runtime.

NeoSpark314 commented:
@hankedan000 in the toolkit I have helper functions to query it: https://github.com/NeoSpark314/godot_oculus_quest_toolkit/blob/34c43c9f89a4b3665acb9fb1da990ab90354fb94/OQ_Toolkit/vr_autoload.gd#L616
